Components/LandingPage/LandingContact.tsx

```diff
@@ -1,6 +1,31 @@
 import Image from "next/image";
+import { getDatabase, ref, push, set } from "firebase/database";
+import { useState } from "react";
 
 const LandingContact = () => {
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [message, setMessage] = useState('');
+
+  function writeUserData(name: any, email: any, message: any) {
+    const db = getDatabase();
+    const usersRef = ref(db, 'users');
```

Comment:
change database name to contact
```suggestion
    const usersRef = ref(db, 'contact');
```